pkg/events/parse_args.go

```diff
@@ -56,7 +56,7 @@ func ParseArgs(event *trace.Event) error {
 				ParseOrEmptyString(prevProtArg, mmapProtArgument, nil)
 			}
 		}
-	case SysEnter, SysExit, CapCapable, CommitCreds, SecurityFileOpen, TaskRename, SecurityMmapFile, KallsymsLookupName, DoMmap:
+	case SysEnter, SysExit:
```

Comment:
Just a question:
So these events will have `context.Syscall="enter"/"exit"` and their argument will be the actual called syscall right?

Reply:
No, the argument and the context value will be matching.
We kept it as is because we want them to be printed in the Table Printer.

Reply:
Wouldn't it be better to add a Syscall column instead? Actually, now all events with syscall arguments will be missing the syscall in the table printer.

Reply:
You are correct, but the `syscall` argument is used in more complicated scenarios so in those cases I think using `json` or `gob` is acceptable.
I think we didn't want to make the Table View more packed as it has too many values to print as it is.